docs/patterns/tile/patterns/accented-tile.html

```diff
@@ -4,6 +4,8 @@ <h2 slot="headline"><a href="#top">Link</a></h2>
 </rh-tile>
 
 <style>
+@import url('/assets/packages/@rhds/elements/elements/rh-tile/rh-tile-lightdom.css');
```

Comment:
pattern files shouldn't have any uxdot-specific stuff (i.e. `/assets/packages/@rhds/elements/elements/...`) in them

I realize there might not be a good workaround in this case. We'll be able to solve it once-and-for all in #1801

Reply:
It's fine